justfile

```diff
@@ -83,12 +87,15 @@ rpc-tests:
     --exclude "sqllogictests/functions/postgres"
 
 #  Check formatting.
-fmt-check: protoc
+fmt-check: protoc requirements
   cargo +nightly fmt --check
+  {{VENV_BIN}}/ruff check .
```

Comment:
I have a (very small) amount of worry that this will make the CI results a bit more confusing.

I also.... don't know that I'd block a build because of a formatting error in the (non-bindings) python code? It's probably fine, and I think having a formatter around is good, but I also wouldn't care if it didn't block CI.